### PR TITLE
feat(addon-table): reorder add possibility to transfer a template

### PR DESCRIPTION
**Pull request by 40oleg**
Closes https://github.com/taiga-family/taiga-ui/issues/6914

### CODE REVIEW

**Comment by 40oleg** [comment]
I see that the text moved a little. I will try to fix it a bit later

File: projects/addon-table/components/reorder/reorder.component.ts [hunk 1]
```diff
@@ -29,6 +30,9 @@ export class TuiReorderComponent<T> {
     @Output()
     public readonly enabledChange = new EventEmitter<T[]>();
 
+    @Input()
+    public container?: PolymorpheusContent;
```
Comment (waterplea) [inline projects/addon-table/components/reorder/reorder.component.ts:34]: We call these things `content`. And let's type context: `PolymorpheusContent<TuiContext<T> & { index: number }>`

File: projects/addon-table/components/reorder/reorder.template.html [hunk 2]
```diff
@@ -15,7 +15,15 @@
                     class="t-icon"
                     [src]="options.icons.drag"
                 />
-                {{ item }}
+                <ng-container
+                    *polymorpheusOutlet="
+                        container ? container : primitiveValue as template;
```
Comment (waterplea) [inline projects/addon-table/components/reorder/reorder.template.html:20]: No need to fallback, you can just add default content as a function that stringifies $implicit: `({$implicit}) => String($implicit)`

File: projects/addon-table/components/reorder/reorder.component.ts [hunk 3]
```diff
@@ -44,6 +46,11 @@ export class TuiReorderComponent<T> {
         }
     }
 
+    @Input()
+    public content?: PolymorpheusContent<TuiContext<T> & {index: number}> = ({
```
Comment (waterplea) [inline projects/addon-table/components/reorder/reorder.component.ts:50]: We need to drop `?:`, otherwise LGTM

**Review by waterplea — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Comment by 40oleg** [comment]
I restarted pipelines because e2e-test for calendar failed for some reasons. I think this is because branch was outdated

**Pull request merged**